### PR TITLE
DOCSP-44969: Modify insert usage examples

### PR DESCRIPTION
**Pull request by norareidy**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-rust/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-44969
Staging - 
https://deploy-preview-154--docs-rust.netlify.app/usage-examples/insertOne/
https://deploy-preview-154--docs-rust.netlify.app/usage-examples/insertMany/

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [ ] Are all the links working?
- [ ] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/usage-examples/insertOne.txt [hunk 1]
```diff
@@ -47,6 +57,7 @@ see the corresponding code for each runtime:
 
          .. input:: /includes/usage-examples/code-snippets/insert-one-async.rs
             :language: rust
+            :emphasize-lines: 19, 35
```
Comment (stephmarie17) [inline source/usage-examples/insertOne.txt:60]: [n] In the deploy preview, I don't see the expected lines highlighted so these may need to be adjusted.

File: source/usage-examples/insertOne.txt [hunk 2]
```diff
@@ -63,6 +74,7 @@ see the corresponding code for each runtime:
 
          .. input:: /includes/usage-examples/code-snippets/insert-one-sync.rs
             :language: rust
+            :emphasize-lines: 18, 34
```
Comment (stephmarie17) [inline source/usage-examples/insertOne.txt:77]: Same as above.

**Review by stephmarie17 — APPROVED** [review]
LGTM with a small fix ✅

**Review by abr-egn — APPROVED** [review]
LGTM!

**Pull request merged**